Tests/SuperLinq.Test/ToListTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Test;
```

Comment:
Additional test should be added to confirm that exception will be thrown when length is <= 0.